.github/workflows/test.yaml

```diff
@@ -17,7 +21,7 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
```

Comment:
Does it matter if we keep `@v2` for tests here if we're changing it to `@v4` in `release-please`, or should this match?

Reply:
Nevermind; I see https://github.com/shaka-project/generic-webdriver-server/pull/106/files already changes it to `@v4`.